src/commands/createNewProject/dotnetSteps/DotnetRuntimeStep.ts

```diff
@@ -28,6 +28,14 @@ export class DotnetRuntimeStep extends AzureWizardPromptStep<IProjectWizardConte
         } else if (runtimes.length === 1) {
             // No need to prompt if it only supports one
             setWorkerRuntime(context, runtimes[0]);
+        } else if (context.targetFramework) {
+            // if a targetFramework was provided from createNewProject
+            const workerRuntime = runtimes.find(runtime => runtime.targetFramework === context.targetFramework);
+            if (!workerRuntime) {
+                throw new Error(localize('unknownFramework', 'Unrecognized target framework "{0}". Available frameworks: "{1}".', context.targetFramework, runtimes.join(', ')));
```

Comment:
Does `runtimes.join` work? I thought they were objects, not strings

Reply:
Yeah, you're right, looks like I'll have to use a .map to get the frameworks.

Reply:
![image](https://user-images.githubusercontent.com/5290572/115312040-7d1f5780-a125-11eb-8689-1cf58c77ff40.png)